src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -731,4 +731,8 @@ protected void verifyNumberOfEmailsSent(Action action, int emailCount) {
         assertEquals(emailCount, action.getEmailSender().getEmailsSent().size());
     }
 
+    protected String getPageResultDestination(String viewUri, boolean error, String userId) {
+        return viewUri + "?error=" + error + "&user=" + userId;
```

Comment:
@leeyimin Should i make parameter `error` a string instead of boolean in order to use `addParamToUrl` .
Also  any suggestions for renaming `viewUri`  : `showPage` or `parentUrl` . Sorry i am bad with names :(